lib/miq_memcached.rb

```diff
@@ -103,10 +110,9 @@ class Control
     #  -S            Turn on Sasl authentication
 
     # TODO: Expose all of the constants and get them from the config
```

Comment:
should this TODO comment go away with the deletion of the last constant below?

Reply:
I thought this comment was referring to exposing the port/user/memory/etc... as reader methods here so I left it, but maybe it meant something else